rackcity/utils/log_utils.py

```diff
@@ -37,6 +37,8 @@ def log_action(user, related_element, action):
     if isinstance(related_element, Asset):
         element_type = ElementType.ASSET.value
         element_name = str(related_element.asset_number)
+        if (related_element.hostname):
+            element_name += (' (' + related_element.hostname + ')')
```

Comment:
including hostname on asset logs so it's more obvious when you're searching by hostname